src/constructs/iam/policies/s3-get-object.ts

```diff
@@ -18,9 +19,14 @@ export class GuGetS3ObjectsPolicy extends GuAllowPolicy {
 }
 
 export class GuGetDistributablePolicy extends GuGetS3ObjectsPolicy {
-  constructor(scope: GuStack, id: string = "GetDistributablePolicy", props?: GuNoStatementsPolicyProps) {
-    const path = [scope.stack, scope.stage, scope.app, "*"].join("/");
-    super(scope, id, { ...props, bucketName: new GuDistributionBucketParameter(scope).valueAsString, paths: [path] });
+  // eslint-disable-next-line custom-rules/valid-constructors -- TODO be better
```

Comment:
😀

Reply:
Yeah, I'm not sure if this is the best change as it means the constructor for the constructs take different forms and goes against [this decision](https://github.com/guardian/cdk/blob/main/docs/architecture-decision-records/002-component-constuctors.md).